portal/patient/styles/style.css

```diff
@@ -299,4 +299,29 @@ span.help-inline {
 /* fixes spacing glitch with combo-box followed by an empty inline help */
 span.help-inline:empty {
 	display: none;
-}
```

Comment:
would it make sense to combine register.css with this?
(this would definitely make it much easier for the international folks to support RTL)

Reply:
Nope register stands alone where styles is used all over framework. Eventually something is going to get stepped on.